languages/ruby/exercises/concept/ostruct/.docs/introduction.md

```diff
@@ -0,0 +1,33 @@
+Ruby comes with a Standard Library - a collection of classes for working with things such as dates, json, and networking.
+The also provide some useful functionality for making your code easier to work with.
```

Comment:
```suggestion
It provides some useful functionality for making your code easier to work with.
```

Reply:
```suggestion
It also provide some useful functionality for making your code easier to work with.
```

(I want the "also" as it's an addition to the classes I mention above), but the The -> It is a good spot, thanks :)

Reply:
```suggestion
It also provides some useful functionality for making your code easier to work with.
```

"it" then makes the "provide" plural.

Reply:
This is "outdated" will check the current version.